packages/commerce-events-collectors/src/utils/product.ts

```diff
@@ -23,4 +23,33 @@ const createPricing = (ctx: Product): ProductPricing | undefined => {
     return pricing;
 };
 
-export { createPricing };
+const createProductFromCartItem = (ctx: ShoppingCartItem): ProductContext => {
+    const pricing : ProductPricing = {
+        regularPrice: ctx.prices.price.value,
+        currencyCode: ctx.prices.price.currency || null,
+    };
+    let productId : any = ctx.product.productId;
```

Comment:
There was some weird issue where productId would be a string, and so needed to parse it, but due to types I have to explicitly set it to any before being able to parse it.

Reply:
yikes

i _think_ you might be able to do `let productId : string = ctx.product.productId as string` but... what do i know... ts continues to surprise me